scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -0,0 +1,337 @@
+<#
+    .SYNOPSIS
+        It is used to streamline running E2E tests for Windows.
```

Comment:
Instead of "It is used", suggest this rewrite: Powershell utility to streamline...